src/runtime-rs/crates/resource/src/rootfs/nydus_rootfs.rs

```diff
@@ -124,6 +130,72 @@ impl NydusRootfs {
                     rootfs_guest_path,
                 ))
             }
+            NYDUS_ROOTFS_V6 => {
+                // enable dax flag for bootstrap and blobfiles.
+                let dax_threshold_size_kb: u64 = 0;
```

Comment:
Why `dax_threshold_size_kb` with hardcode `0` ?

One more question but important: does it need to check DAX whether it's enabled ?